### ISSUE TITLE
The endpoints

### ISSUE THREAD

**Issue by hodamansour** [lotus-1/OnlineTestClasses#2]
/home
render the home page,
with two buttons: log-in and sing-up.
/login
render the login page,
with form that have two inputs : user name and password.
/sing-up
render the sing-up page,
with form that have 4 inputs: name, email, password and confirm password.
/classes
render the classes page
with 3 button and every button have a list for 3 endpoints: /Arabic , /Hebrew, /English 
(when we click the button open a list<ul> every row<li> open endpoint)

**Comment by Jasmineb2011** [comment]
I don't think that with each button in the classes page for opening lists will contain endpoints, because basically we're still in the same page.
